Scarb.toml

```diff
@@ -7,8 +7,9 @@ version = "0.6.0"
 sierra-replace-ids = true
 
 [scripts]
-migrate = "sozo build && sozo migrate && ./scripts/default_auth.sh"
+migrate = "sozo build && sozo migrate plan && sozo migrate apply && ./scripts/default_auth.sh"
```

Comment:
since profile is used to generate manifests path, we could make it explicit with `[profile.dev.tool.dojo.env]` and in commands `sozo -P dev build && ..` 
it give an example for how to use profiles too

Reply:
Also we don't need to execute plan before apply. I'll document that. Migrate is doing all that plan does + applying the diff on-chain. 👍

Reply:
Awesome. Fixed on commit [adfbec0](https://github.com/dojoengine/dojo-starter/pull/52/commits/adfbec0f56c26d3bc27517b4423e9def63aee497). Please check

Reply:
Quick question, usually people don't use scarb when they use dojo. I'm wondering if running `scarb migrate` is something we want to do.

IMHO to not mix, I would prefer a bash script than a scarb script. What's your take on that? For sure scarb scripting is here for that.. so why not using it. Just wondering how much it can confuse the user. :)

Reply:
@ponderingdemocritus @notV4l